### DISCUSSION TITLE
macOS self-build, possible to use output of `make package` without signature?

### DISCUSSION THREAD

**Discussion by bsl** [keepassxreboot/keepassxc#11475] [Q&A]
If we build ourselves on macOS with `make package`, we'll get a thing we can drag and drop into Applications, but that thing won't run due to not being signed, and I was unable to find any way to run it. Is there a way?

**Comment by droidmonkey** [comment]
Usually you can right click and select open, then you can allow a bypass

**Reply by bsl** [reply]
Thanks for the suggestion. If I right click and select open, I get `KeePassXC quit unexpectedly.` Here's the relevant stuff from the report:

```
Process:               KeePassXC [98689]
Path:                  /Applications/KeePassXC.app/Contents/MacOS/KeePassXC
Identifier:            org.keepassxc.keepassxc
Version:               2.8.0 ()
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501
OS Version:            macOS 15.1 (24B83)
System Integrity Protection: enabled
Exception Type:        EXC_BAD_ACCESS (SIGKILL (Code Signature Invalid))
```

**Reply by bsl** [reply]
I found something that allows it to run. `codesign --force --deep --sign - /Applications/KeePassXC.app`
